### ISSUE TITLE
Difficulty getting started with memote

### ISSUE THREAD

**Issue by artgoldberg** [opencobra/memote#272]
#### Difficulty getting started

Overview:
* I want to learn about Memote
* I installed and ran it on your test model, and read some documentation but am unable to clearly evaluate Memote

Issues:
* memote runs, but the stdout and stderr output is difficult to understand and the index.html file generated is empty
* in the code, memote/suite/reporting/reports/snapshot.py, SnapshotReport.render_html() returns an empty string
* 'memote -h' and 'memote  report -h' could produce more comprehensive documentation. E.g., many lines are truncated like this
```
  history   Generate a report over a model's git commit...
  snapshot  Take a snapshot of a model's state and...
```
Alternatively, https://memote.readthedocs.io/ could be more comprehensive.

#### Commands
```
git clone https://github.com/opencobra/memote.git
memote report snapshot tests/data/EcoliCore.xml.gz
ls -l index.html
memote -h
memote  report -h

```
Thanks
Arthur

### Context

<details>
>>> memote.show_versions()

System Information
==================
OS                Darwin
OS-release        14.5.0
Python             3.5.4

Package Versions
================
pip                9.0.1
setuptools        36.2.0
cobra              0.9.1
future            0.16.0
swiglpk            1.4.4
optlang            1.3.0
ruamel.yaml      0.14.12
pandas            0.20.3
numpy             1.13.1
python-libsbml    5.15.0
lxml               3.6.4
click                6.7
click-configfile   0.2.3
click-log          0.2.1
pytest             3.3.0
dask              0.16.0
cloudpickle        0.5.2
toolz              0.8.2
Jinja2             2.9.6
jinja2-ospath      0.2.1
cookiecutter       1.6.0
plotly             2.2.2
</details>

**Comment by Midnighter** [comment]
Dear Arthur,
Thank you for trying out memote and reporting on your difficulties here. I'll try to address them point-by-point.

> memote runs, but the stdout and stderr output is difficult to understand and the index.html file generated is empty

Our default branch on GitHub is "develop" which you installed. We are currently changing the way reports are generated from being jinja2 template based to an Angular app. Since we're also changing around the structure quite a bit, I have disabled the HTML reports until the Angular app is in. You should be able to see the old version of the HTML report if you installed memote 0.4.6 directly from PyPi (`pip install memote==0.4.6`).

Although reading messages on the command line is never fun, I'm saddened to hear that you found the messages difficult to understand because we actually but some effort into making them clearer and more informative. Do you have any recommendation for how they could be improved?

So the slight dilemma is that 0.4.6 has the old HTML report which makes the overview better but the develop branch has the "improved" messages but no report. We are aiming for a next memote release with the Angular report by the end of next week. May I ask you to try again with the next release (probably 0.5.0) when it's out?

> 'memote -h' and 'memote report -h' could produce more comprehensive documentation. E.g., many lines are truncated

The truncation is a "feature" of click. You can see the full text from the command line if you ask for the help on the final sub command, e.g., `memote report snapshot -h`. I agree that the documentation could use a lot of TLC. We will focus on documentation when we have stabilized the major moving parts of memote.

**Comment by Midnighter** [comment]
Dear @artgoldberg, a bit later than promised but we just released memote 0.5.1 with much more extensive test case explanations and a visually much more pleasing HTML report. Do check it out, please. Feel free to re-open this issue if there's something missing in your opinion.
